source/Tools/Playnite.Toolbox/Playnite.Toolbox.csproj

```diff
@@ -71,10 +71,6 @@
       <Link>Templates\Themes\Fonts\icofont.ttf</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Include="..\..\..\references\Fonts\PlayStation4.ttf">
-      <Link>Templates\Themes\Fullscreen\Fonts\PlayStation4.ttf</Link>

```

Comment:
Really? Just removing files from a project without any comment or anything?

Reply:
Oof. My bad. Actually, *you* deleted the file! Check your repo -- the files don't exist. It just hadn't been removed from the .csproj, and the fact that it still shows up in .csproj breaks any build done from scratch.  So either the file needs to be readded or the .csproj does need to be changed.  But *neither* one of those obviously belongs in this pull request. I thought I'd branched off those changes separately but apparently I didn't.

Reply:
https://github.com/JosefNemec/Playnite/wiki/Building#fullscreen-mode-development

This applies to Toolbox as well since it's handling Fullscreen templates.

Reply:
Link for the resources in that page leads to a page that says:
"Unfortunately, Xbox One & PlayStation 4 Buttons Pack is no longer available."

Perhaps the right solution, then, is to take Toolbox out of the default build and let folks enable it if they need it.  Probably beats breaking a default build from a fresh Git pull, I'd think.  (And the dependency of Toolbox on those font assets should also be mentioned in the build doc; Toolbox is never mentioned in the build doc at all, and I know I haven't even used Toolbox and have never even had the need to find out what it does, so I definitely wouldn't know that it's "handling Fullscreen templates" and wouldn't have any way to directly find out.  In the end, the fact that a from-scratch clean build would break due to a dependency on a dev tool I've never used nor needed to would seem to imply *something* there needs fixing one way or the other.)

Regardless, this has been pulled out of this PR.  It certainly doesn't belong here.